packages/@webex/plugin-meetings/src/multistream/remoteMediaGroup.ts

```diff
@@ -66,6 +67,50 @@ export class RemoteMediaGroup {
     return [...this.unpinnedRemoteMedia, ...this.pinnedRemoteMedia];
   }
 
+  /**
+   * setActiveSpeakerCsis - sets csis for remoteMedia
+   * pins when csis is there and unpins when csi is undefined
+   */
+  public setActiveSpeakerCsis(
+    remoteMediaCsis: {remoteMedia: RemoteMedia; csi?: number}[],
+    commit = true
+  ): void {
+    forEach(remoteMediaCsis, ({csi, remoteMedia}) => {
```

Comment:
any reason why we're using `forEach` from lodash instead of javascript's Array.forEach() - so just `remoteMediaCsis.forEach()`?

Reply:
Reads better for me